internal/converter/converter_test.go

```diff
@@ -73,7 +80,7 @@ func testConvertSampleProto(t *testing.T, sampleProto sampleProto) {
 		t.Fail()
 	} else {
 		for responseFileIndex, responseFile := range response.File {
-			assert.Equal(t, sampleProto.ExpectedJSONSchema[responseFileIndex], *responseFile.Content, "Incorrect JSON-Schema returned for sample proto file (%v)", sampleProtoFileName)
+			assert.Equal(t, strings.TrimSpace(sampleProto.ExpectedJSONSchema[responseFileIndex]), *responseFile.Content, "Incorrect JSON-Schema returned for sample proto file (%v)", sampleProtoFileName)
```

Comment:
I added this because after moving `c.parseGeneratorParameters` under `convert`, the expected schema contained a newline at the end